log/logtest/factory_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package logtest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/exp/slices"
```

Comment:
We can use the standard library

```suggestion
	"slices"
```